packages/build-tools/src/utils/outputs.ts

```diff
@@ -0,0 +1,81 @@
+import { Generic } from '@expo/eas-build-job';
+import { BuildStep, jsepEval } from '@expo/steps';
+import { bunyan } from '@expo/logger';
+import nullthrows from 'nullthrows';
+
+import { BuildContext } from '../context';
+
+import { turtleFetch } from './turtleFetch';
+
+export async function uploadJobOutputsToWwwAsync(
+  ctx: BuildContext<Generic.Job>,
+  {
+    steps,
+    logger,
+    expoApiV2BaseUrl,
+  }: { steps: BuildStep[]; logger: bunyan; expoApiV2BaseUrl: string }
+): Promise<void> {
+  if (!ctx.job.outputs) {
+    logger.info('Job defines no outputs, skipping upload');
+    return;
+  }
+
+  try {
+    const workflowJobId = nullthrows(ctx.job.builderEnvironment?.env?.__WORKFLOW_JOB_ID);
+    const robotAccessToken = nullthrows(ctx.job.secrets?.robotAccessToken);
+
+    const interpolationContext = {
+      steps: Object.fromEntries(steps.map((step) => [step.id, getDynamicValuesFromStep(step)])),
+    };
+    logger.info({ dynamicValues: interpolationContext }, 'Using dynamic values');
```

Comment:
I believe that parts of the logs that are not part of the `msg` field are not shown on the website.
```suggestion
    logger.info(`Using dynamic values: ${JSON.stringify(interpolationContext)}`);
```

Reply:
You are right that it is not visible. However, now that I think on it, I'm not sure if it should be printed to the user. It's been copied from the existing implementation in Turtle, which didn't print anything to the user, but this particular line looks to me as better suited to be printed in the non-user-facing logs only, wdyt?

Reply:
yeah, we can add it to non-user-facing logs